examples/OrmliteExample/app/src/main/res/menu/menu_main.xml

```diff
@@ -0,0 +1,6 @@
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I would remove all things not needed, like all sized pictures, menus, styles, etc. So keep the example as mall as clean, and concentrate on what you demonstrate.